### PR TITLE
Show a notification when getting logged out

### LINKED ISSUE #355: Show reason for logging a user out/prompting for a login
If you have the Now desktop client running and remove any of the configuration files, Now will (correctly) prompt you to login again.

However, the reasoning for it isn't immediately clear, and in the event it's _unexpected_, it could be frustrating to be interrupted - especially if there's a rogue script removing the configuration over and over for some reason.

We should include a logout reason and/or notification for being logged out whenever these types of things occur.

There is currently the `logout` function that takes no arguments. However, I think it should take at least a `reason` argument that is displayed to the user.

Further, I don't know that we should immediately pop up the window for logging in again since the missing configuration could be temporary - many users choose to mount their home directory on some external media that may become temporarily unavailable from time to time.

If/when this is fixed, we should make sure the fix accounts for boot-ups - that is, the login window _should_ automatically display if the configuration is missing upon starting up Now Desktop.

### PR DESCRIPTION
**Pull request by leo**
This closes #355.

screen shot 2017-09-11 at 17 43 16

### CODE REVIEW

**Comment by Qix-** [comment]
Can we add "Click here to log in again" to the end and add a handler for clicking on it to bring up the login dialog?

**Comment by leo** [comment]
@Qix- 👍 Added

File: main/utils/logout.js [hunk 1]
```diff
@@ -98,15 +99,46 @@ module.exports = async () => {
...
+      let body
+
+      // This can be extended later
+      switch (reason) {
+        case 'config-removed':
+          body =
+            'You were logged out from Now Desktop because you logged out from Now CLI.'
+          break
+        default:
+          body = false
```
Comment (TooTallNate) [inline main/utils/logout.js]: necessary? The `undefined` value in `body` will make the if check behave properly still.

Comment (leo) [reply main/utils/logout.js]: XO requires a default value... 😄

**Pull request merged**